bpfd/src/bpf.rs

```diff
@@ -185,7 +198,10 @@ impl BpfManager {
     ) -> Result<Uuid, BpfdError> {
         debug!("BpfManager::add_multi_attach_program()");
 
-        let program_bytes = program.data_mut().program_bytes().await?;
+        let program_bytes = program
```

Comment:
Nit can you move this into add_program() so that we don't have to call the same code twice in add_multi && add_single etc